src/frontend/src/showcase.ts

```diff
@@ -227,7 +227,11 @@ const iiPages: Record<string, () => void> = {
   chooseDeviceAddFlow: () => chooseDeviceAddFlow(),
   renderPollForTentativeDevicePage: () =>
     renderPollForTentativeDevicePage(userNumber),
-  addRemoteDevice: () => addRemoteDevice(dummyConnection),
+  addRemoteDevice: () =>
+    addRemoteDevicePage({
```

Comment:
you can safely ignore the promise result here?

Reply:
yes, it's just a showcase! the result is meaningless